ui/src/modals/newtask.vue

```diff
@@ -264,35 +264,54 @@ export default {
             });
 
             //now find apps that user can submit
-            this.$http.get('app', {params: {
-                find: JSON.stringify({
-                    "inputs.datatype": {$in: datatype_ids},
-                    removed: false,
-                }),
+            this.$http.get('/app/query', {params: {
```

Comment:
better keep the datatypes in the query, and handle it on the backend.
as discussed, we need to figure it out in the backend the apps that are compatible or not. right now, all the apps will be returned.